style.css

```diff
@@ -0,0 +1,640 @@
+* {
+  margin: 0% 0%;
+  padding: 0% 0%;
+  overflow-x: hidden;
+}
+
+.container {
+  display: grid;
+  height: 130rem;
+}
+
+
+
+
+
+
+/* HEADER - Trying implement BEM-tags for block, element and modifier*/
+
+
+.header-block {
+  grid-area: header;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.header-block__audio {
+  position: absolute;
+  width: 25vw;
+  margin-left: 75vw;
+  align-self: baseline;
+  border: solid 0.1px rgb(237, 243, 243);
+  background-color: rgb(237, 243, 243);
+}
+
+.header-block__video {
```

Comment:
Cool! I will try to implement this in future projects (haven't tried using Z-index before). :)